modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsControllerEpics.scala

```diff
@@ -389,12 +389,21 @@ object TcsControllerEpics extends TcsController {
       case Subsystem.ScienceFold => setScienceFoldPosition(tcs.agc.sfPos)
     }
 
-    subsystems.tail.foldLeft(configSubsystem(subsystems.head))((b, a) => b *> configSubsystem(a)) *>
+    val overrideHRProbe: Boolean = tcs.agc.sfPos.flatMap{
```

Comment:
I think this could be simplified with `collect`

Reply:
I would agree, if it weren't for `getInstPort` returning an `Option`.